Assignment/oddeven.txt

```diff
@@ -0,0 +1,21 @@
+/* [16] WAP to check number is Even or Odd
+ID : 23BCA275 */
+
+#include<iostream.h>
+#include<conio.h>
+void main()

```

Comment:
why you using void main() in c++, c++ refers to use int main() and its return a vallue.

Reply:
ok thanks